modules/microshift-config-yaml.adoc

```diff
@@ -6,17 +6,22 @@
 [id="microshift-config-yaml_{context}"]
 = Using a YAML configuration file
 
-At start up, {microshift-short} checks the system-wide `/etc/microshift/` directory for a configuration file named `config.yaml`. If the configuration file does not exist in the directory, the built-in default values are used to start the service.
+At start up, {microshift-short} checks the system-wide `/etc/microshift/` directory for a configuration file named `config.yaml`. If the configuration file does not exist in the directory, built-in default values are used to start the service.
 
 [id="microshift-yaml-custom_{context}"]
 == Custom settings
-To create custom configurations, you must create a `config.yaml` file in the `/etc/microshift/` directory, and then change any settings that are expected to override the defaults before starting or restarting {microshift-short}.
+To create custom configurations, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`. Keep this file in the `/etc/microshift/` directory, and then you can change supported settings that are expected to override the defaults before starting or restarting {microshift-short}.
```

Comment:
Just to make it clear that the file you need to copy is in the specified directory. Might be overkill though.
```suggestion
To create custom configurations, make a copy of the provided `config.yaml.default` file that is in the `/etc/microshift/` directory, renaming it `config.yaml`. Keep this file in the `/etc/microshift/` directory, and then you can change supported settings that are expected to override the defaults before starting or restarting {microshift-short}.
```